ReactNativeClient/lib/joplin-renderer/MdToHtml.js

```diff
@@ -161,7 +184,6 @@ class MdToHtml {
 					context.pluginAssets['highlight.js'] = [
 						{ name: options.codeTheme },
 					];
-
```

Comment:
The commit would be much easier to read if it wasn't for all these unnecessary white space changes.